msteams-platform/tabs/how-to/using-teams-client-sdk.md

```diff
@@ -32,18 +32,18 @@ The following table lists Teams tabs and dialogs (task modules) capabilities (pu
 
 |Capability | Host support | Notes |
 |-----------|--------------|-------|
-| app | Teams, Outlook, Office | Namespace representing app initialization and lifecycle. |
+| app | Teams, Outlook, Office, Office for Android | Namespace representing app initialization and lifecycle. |
```

Comment:
do we want to be explicit for Outlook platforms (web, OWA, Android and iOS)?

Reply:
Yes, we will add iOS upon preview